src/main/java/me/partlysanestudios/partlysaneskies/chat/ChatManager.kt

```diff
@@ -51,9 +55,12 @@ object ChatManager {
             messageToSend = ChatAlertsManager.checkChatAlert(messageToSend)
         }
 
+        messageToSend = ChatComponentText(StringUtils.colorCodes(WordEditor.handleWordEditorMessage(messageToSend.unformattedText)));
```

Comment:
This should be protected in an if statement

```kt
if (WordEditor.shouldEditMessage(messageToSend)) {
    /* current code*/
}
```

Reply:
oh